test/fixtures/shared/variables.tf

```diff
@@ -14,29 +14,33 @@
  * limitations under the License.
  */
 
+variable "project_id" {
+  description = "The project ID"
+  type        = string
+}
+
 variable "parent_id" {
   description = "The parent of this AccessPolicy in the Cloud Resource Hierarchy. As of now, only organization are accepted as parent."
+  type        = string
 }
 
 variable "policy_name" {
   description = "The policy's name."
+  type        = string
 }
 
 variable "protected_project_ids" {
   description = "Project id and number of the project within the regular service perimeter"
-  type        = "map"
+  type        = object({ id = string, number = number })
 }
 
 variable "public_project_ids" {
   description = "Project id and number of the project OUTSIDE of the regular service perimeter. This variable is only necessary for running integration tests. This map variable expects an \"id\" for the project id and \"number\" key for the project number."
-  type        = "map"
+  type        = object({ id = string, number = number })
+  default     = null
 }
 
 variable "members" {
-  type    = "list"
-  default = []
-}
-
-variable "credentials_path" {
-  description = "Path to credentials.json key for service account deploying resources"
+  type    = list(string)
+  default = null
```

Comment:
IMHO will be better to use as default [] for list

Reply:
Thank you, Ihor!
[Resolved](https://github.com/terraform-google-modules/terraform-google-vpc-service-controls/commit/63c5a8a98dbafd8436547a6d994293cafb99c825)